### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Al1243** [termux/termux-app#3388] [Q&A] [answered]
Why does the program tell me to unable to locate the package when I want to install a package? i use Android 13 my device is Xiaomi Poco x3 pro

**Comment by sylirre** [comment, marked as answer]
Use `pkg update` to refresh packages list.

Pay attention to this:
* Termux does not provide development headers (-dev packages) separately. For example if you need `libcurl-dev`, you'll have to install `libcurl`.
* Termux does not provide software suitable solely for hacking. We deleted previously available packages like `hydra` from our repositories.
* Many package names are not same as on Debian.
* Certain packages are available only if `root-repo` or `x11-repo` installed.
* `unstable-repo`, `game-repo`, `science-repo` merged with main repository and no longer available.
* Double-check whether you are typing the package name correctly. For example `python` is the correct package to install Python v3.x, not `phyton` (commonly seen mistake).
